.github/workflows/_conformance_tests.yaml

```diff
@@ -6,6 +6,12 @@ on:
 jobs:
   conformance-tests:
```

Comment:
Instead of using a matrix for this test case, I would create two different use cases to improve readability: With this setting, the GH action output is not clear:
```
PR checks / conformance-tests / conformance-tests (false) (pull_request) Successful in 7m Details
PR checks / conformance-tests / conformance-tests (false) (pull_request) Successful in 6m Details
```

While the following output would be way clearer IMO:

```
PR checks / conformance-tests / conformance-tests-traditional-compatible-router Successful in 7m Details
PR checks / conformance-tests / conformance-tests-expression-router Successful in 6m Details
```

WDYT?

Reply:
+1. I would suggest using the same pattern as we have in `_integration_tests.yaml`. `matrix.include` with testcases + job's name set to `${{ matrix.name }}`. This way we can avoid duplication of the job's code while still have the expected explicit names.

Reply:
done.